### ISSUE TITLE
Need to dec-to-hex function to manipulate color strings, particularly in IE

### ISSUE THREAD

**Issue by heygrady** [Compass/compass#259]
You can see that [madr](https://github.com/madr/css3-sass-mixins/) is trying to make a SASS/Compass mixin that supports everything in [CSS3 Please!](http://css3please.com/) and is unable to because SASS doesn't allow for string manipulation on colors and doesn't support a dec-to-hex function.

Here's an issue he's filed on his own library.
https://github.com/madr/css3-sass-mixins/issues#issue/2

The idea is that you could create an ie-rgba mixin like what is shown below. A dechex function is just about as practical as the red, green and blue functions that already exist in SASS.

```
@mixin ie-rgba($red, $green, $blue: undefined, $alpha: undefined) {     
    // using short-hand of ie-rgba(hex, alpha)
    @if $blue == undefined {
        $alpha: $green;
        $color: $red;
        $red: red($color);
        $green: green($color);
        $blue: blue($color);
    } 

    // convert to an IE alpa color of form AARRGGBB
    $alphacolor: '#' + dechex(($alpha * 255)) + dechex($red) + dechex($green) + dechex($blue);

    // create a filter string
    $filter: progid:DXImageTransform.Microsoft.gradient(startColorStr='#{$alphacolor}', EndColorStr='#{$alphacolor}');

    // IE6 and 7 don't need it quoted
    .ie7 &, .ie6 & {
        background: none;
        filter: $filter;
    }

    // IE8 prefers the quoted filter and the vendor prefixed property
    .ie8 & {
        background: none;
        -ms-filter: quote($filter);
    }
}
```

**Comment by chriseppstein** [comment]
I already made this function to support the [filter-gradient mixin](https://github.com/chriseppstein/compass/blob/master/frameworks/compass/stylesheets/compass/css3/_images.scss#L50-63). It is `ie-hex-str($color)`. The color can be in any form that sass understands and it will get translated to the backward ie hex string they use.

This feature is available in the latest betas.

**Comment by heygrady** [comment]
My guess is that this isn't fully released yet? I just ran a gem update and this doesn't appear to work for me. I don't have any of the edge stuff installed.

**Comment by chriseppstein** [comment]
It's in beta and will be released once Sass 3.1 is out. You can install the beta like so:

```
gem install compass --pre
```
